sherpa/astro/ui/tests/test_astro_ui_fluxes.py

```diff
@@ -1154,6 +1176,175 @@ def test_sample_foo_flux_scales_example(multi, make_data_path, clean_astro_ui,
     assert ans[:, 0].min() > 0
 
 
+@requires_data
+@requires_fits
+def test_bug_276(make_data_path):
+    ui.load_pha(make_data_path('3c273.pi'))
+    ui.set_model('polynom1d.p1')
+    ui.fit()
+    ui.covar()
+    scal = ui.get_covar_results().parmaxes
+    ui.sample_flux(ui.get_model_component('p1'), 0.5, 1, num=5,
+                   correlated=False, scales=scal)
+
```

Comment:
Test any result here? The test ensure that it's not throwing an error, but that could fullfilled is `sample_flux` was a no-op. No need to go crazy, just to see if anything happens?

Reply:
This is an old test, which I've just moved file in this PR, so I don't feel the need to work out what useful tests we can add to it.

There are explicit tests added in this PR which test the sample_flux output, so it's not like we don't have tests.